puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_plugin_matrix.yaml

```diff
@@ -44,12 +48,10 @@
             - 2.0.0
             - 2.1
       - axis:
-          type: user-defined
+          type: dynamic
```

Comment:
This looks like it requires an additional plugin, so it'll have to wait until I get an opportunity to install it.